test/test_getting_columns.py

```diff
@@ -200,12 +201,14 @@ def test_columns_with_comments():
     parser = Parser(
         "INSERT /* VoteHelper::addVote xxx */  INTO `page_vote` (article_id,user_id,`time`) VALUES ('442001','27574631','20180228130846')"
     )
+    assert parser.query_type == "Insert"
```

Comment:
I'm wondering if we shouldn't introduce a set of const values (like `sql_metadata.consts.query_types.INSERT`) to ease the use and avoid typos.

Reply:
Was thinking about Enum, wanted your inside :)

Reply:
Can be enum as well - leaving this up to you 🙂

Reply:
Switched to enum